app/pages/collections/list.cjsx

```diff
@@ -50,7 +50,7 @@ List = React.createClass
   listCollections: (props) ->
     query = {}
     if props.params?.collection_owner?
-      query.owner = props.params.collection_owner
+      query.current_user_roles = "owner,contributor,collaborator"
```

Comment:
`if props.params.collection_owner is props.user.display_name` sure seems like it should work, but there's apparently a race condition where that `props.user` object only exists after the user is grabbed but this is rendered before.